Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -153,6 +167,12 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
                                               body:notification.userInfo];
 }
 
+- (void)handleRemoteNotificationRegisteredError:(NSNotification *)notification
+{
+  [_bridge.eventDispatcher sendDeviceEventWithName:@"remoteNotificationsRegisteredError"
+                                              body:[notification userInfo]];
```

Comment:
Judging by the preceding notification handlers, dot-notation is preferred: `notification.userInfo`.